tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.            # This could have been torch.randn(self.X.shape[1], self.hidden_size)
</pre><p>Why is that? Doesn't one returns a <code>dist.Distribution</code>  (the API required for a <code>random_variable</code>)whereas the other returns a <code>torch.Tensor</code> (the API required for a <code>bm.param</code>)?     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Totally correct!